source/code/plugins/operation_lib.rb

```diff
@@ -39,9 +39,28 @@ def filter(record, time)
 
       return {}
     end
+
+    def filter_generic(record,time)
+      if record.is_a?(Hash) && !record.empty? && record.has_key?("message")
+        dataitem = {}
+        dataitem["Timestamp"] = OMS::Common.format_time(time)
+        dataitem["OperationStatus"] = "Warning"
+        dataitem["Computer"] = OMS::Common.get_hostname or "Unknown host"
+        dataitem["Detail"] = record["message"]
```

Comment:
@sugr4 @varunkumta 

- Category = OMS Agent for Linux issue
- Solution = Log Management